store/index.js

```diff
@@ -62,37 +45,6 @@ const whitelist = {
 window.Observable = Observable;
 
 export default function configureStore(initialState) {
-    const setTransformer = createTransform(
```

Comment:
This whole transformation logic was used on mobile to serialize/deserialize Set objects being persisted in the store. The web app doesn't do that, so I removed it for simplicity's sake while mocking redux-persist (see below)